src/main/java/models/Game.java

```diff
@@ -0,0 +1,93 @@
+package models;
+
+import java.lang.reflect.Array;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Game {
+
+    private static final Game GAME_INSTANCE = new Game();
+
+    private static ArrayList<Player> allPlayers;
+    private static ArrayList<Team> allTeams;
+    private ArrayList<League> leagues;
+
+    private Game(){
+        allPlayers = new ArrayList<>();
+        allTeams = new ArrayList<>();
+        leagues = new ArrayList<>();
+    }
+
+    public void addPlayers(Player newPlayer){
+        allPlayers.add(newPlayer);
+    }
+
+    public ArrayList<String> getAllPlayersNames() {
+        return new ArrayList<>(allPlayers.stream().map(player -> player.getName()).collect(Collectors.toList()));
+    }
+
+    public ArrayList<Player> getAllPlayers() {
```

Comment:
A bit confusing as the top function gets all the player's name, what does this function do? Maybe rename to getAllPlayersStats or something similar